app/service.js

```diff
@@ -27,7 +27,15 @@ function getDocumentMetadata(url, window) {
     metadata.favicon_url = parsedUrl.protocol + '//' + parsedUrl.host + '/favicon.ico';
   }
 
-  metadata.image = metadata.image && metadata.image.replace(/^\/\//, 'https://');
+  metadata.image_url = metadata.image_url && metadata.image_url.replace(/^\/\//, 'https://');
+  metadata.images = [{
+    'url': metadata.image_url,
```

Comment:
Here's my results of randomly curled bing.com and duckduckgo.com.

Note that bing.com is kind of choking on a `data://` URI, whereas duckduckgo.com fails differently where we hit the case above of no `image_url` and then get an odd response:

```json
      "favicon_url": "https://duckduckgo.com/favicon.ico",
      "images": [
        {
          "width": 500,
          "height": 500,
          "entropy": 1
        }
      ]
```

```sh
$ curl -XPOST -i -H "content-type: application/json" -d '{"urls": ["http://bing.com", "https://duckduckgo.com"]}' http://localhost:7001 | JSON

HTTP/1.1 200 OK
X-Powered-By: Express
Content-Type: application/json; charset=utf-8
Content-Length: 1050
ETag: W/"41a-xW9Ul4AWhP2zJ1a2DZQGZQ"
Date: Tue, 09 Aug 2016 00:17:10 GMT
Connection: keep-alive

{
  "error": "",
  "urls": {
    "http://bing.com": {
      "description": "Bing helps you turn information into action, making it faster and easier to go from searching to doing.",
      "icon_url": "/fd/s/a/hp/bing.svg",
      "image_url": "data://undefinedimage/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAEALAAAAAABAAEAAAIBTAA7",
      "title": "Bing",
      "url": "http://bing.com",
      "original_url": "http://bing.com",
      "provider_url": "http://bing.com",
      "favicon_url": "http://bing.com/favicon.ico",
      "images": [
        {
          "url": "data://undefinedimage/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAEALAAAAAABAAEAAAIBTAA7",
          "width": 500,
          "height": 500,
          "entropy": 1
        }
      ]
    },
    "https://duckduckgo.com": {
      "description": "The search engine that doesn't track you. A superior search experience with smarter answers, less clutter and real privacy.",
      "icon_url": "/assets/icons/meta/DDG-iOS-icon_60x60.png",
      "title": "DuckDuckGo",
      "url": "https://duckduckgo.com",
      "original_url": "https://duckduckgo.com",
      "provider_url": "https://duckduckgo.com",
      "favicon_url": "https://duckduckgo.com/favicon.ico",
      "images": [
        {
          "width": 500,
          "height": 500,
          "entropy": 1
        }
      ]
    }
  }
}
```